src/SonataCoreBundle.php

```diff
@@ -29,8 +29,15 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new StatusRendererCompilerPass());
         $container->addCompilerPass(new AdapterCompilerPass());
 
+        $formMappingIsEnabled = true;
+        foreach ($container->getExtensionConfig('sonata_core') as $config) {
```

Comment:
Did you test what returns `getExtensionConfig` on an application? I think it does not take into account default values for config, because form mapping is enabled by default right?

Reply:
No, I did not check on a real app. I asked someone who suffered from this deprecation on slack to test. I don't think `Configuration` is taken at all into account indeed. This is the array of arrays that gets passed to the extension (which chooses to use `Configuration` or not to process it)